docs/high/dataset.rst

```diff
@@ -261,6 +261,34 @@ safe to use with very large target selections.  It is supported for the above
    more information.
 
 
+.. warning::
+    Dataset item getter returns new numpy.ndarray. Some might find the following behavior surprising:
```

Comment:
This doesn't quite read right - can you expand it a bit? I'm happy to have a go as a native english speaker if you prefer.